app/middlewares/auth.js

```diff
@@ -36,3 +36,12 @@ exports.verifyPermission = async (req, res, next) => {
     ? next(error.authorizationError('Is not  an Administrator user'))
     : next();
 };
+
+exports.verifyAccess = async (req, res, next) => {
+  const auth = req.headers[AUTHORIZATION];
+
+  const user = await getUser(auth);
```

Comment:
You already obtain user in `verifyToken` middleware, put in some place like `req.user` and use here